src/server/binary/binaryServerStarter.ts

```diff
@@ -112,11 +112,16 @@ async function downloadBinary(): Promise<string> {
       });
   });
   downloadPromise.then((_binaryPath) => {
-    Telemetry.sendTelemetry(Telemetry.BINARY_DOWNLOAD_SUCCEEDED_EVT);
+    const data: any = {};
+    data[Telemetry.BINARY_DOWNLOAD_SUCCESS_PROP] = true;
+    Telemetry.sendTelemetry(Telemetry.BINARY_DOWNLOAD_EVT, data);
   });
   downloadPromise.catch(e => {
     if (e !== ABORTED_ERROR) {
-      Telemetry.sendTelemetry(Telemetry.BINARY_DOWNLOAD_FAILED_EVT);
+      const data: any = {};
+      data[Telemetry.BINARY_DOWNLOAD_SUCCESS_PROP] = false;
+      data['error'] = e;
+      Telemetry.sendTelemetry(Telemetry.BINARY_DOWNLOAD_EVT, data);
```

Comment:
what about aborted events? that's another state. boolean is not sufficient for that

Reply:
How about a `status` parameter with `succeeded`, `failed`, and `aborted` as possible values?

Reply:
yup